docs/background.md

```diff
@@ -12,7 +12,7 @@ The primary goals of treat are **full static extraction, minimal runtime code, t
 
 Unlike a lot of CSS-in-JS approaches, treat is much more similar to [CSS Modules](https://github.com/css-modules/css-modules), requiring a bit more work to bind styles to your components. It's also unable to generate styles at runtime, which means it cannot handle dynamic theming. The upside of treat is that it allows you to craft themeable, statically extracted CSS using JavaScript (or TypeScript) with little impact to bundle size and negligible runtime performance cost.
 
-If you're used to a libraries like [styled-components](https://www.styled-components.com) or [Emotion](https://emotion.sh/), treat might seem like a step backwards. These libraries can quickly and easily create dynamic component-oriented styles. However, they come with a bundle size and performance cost. For plenty of applications, this is a worthwhile tradeoff—and these libraries are a great choice if this isn't an issue for you.
+If you're used to libraries like [styled-components](https://www.styled-components.com) or [Emotion](https://emotion.sh/), treat might seem like a step backwards. These libraries can quickly and easily create dynamic component-oriented styles. However, they come with a bundle size and performance cost. For plenty of applications, this is a worthwhile tradeoff — and these libraries are a great choice if this isn't an issue for you.
```

Comment:
That lack of space around the em dash is actually correct usage: https://www.thepunctuationguide.com/em-dash.html

Reply:
This from the man who can never get enough whitespace 😂 

I'll revert 😄